EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1230,273 +1230,273 @@ QUEST_LV_0300_20160310_001229	Even if you can't trust me you should trust Vaidot
-QUEST_LV_0300_20160310_001269	With this much, you would be able to purify the soul marble.{nl}Please wait a bit.
-QUEST_LV_0300_20160310_001270	As the power of the spirit is strong, the purified power is also strong.{nl}This power would be able to destroy the vicious idol.{nl}
-QUEST_LV_0300_20160310_001271	Put the power of the soul marble into the purified evil idol.{nl}I am sure the idol would not withstand..
-QUEST_LV_0300_20160310_001272	Finally!{nl}Now, we would be free from the black wall.{nl}
-QUEST_LV_0300_20160310_001273	It's been such a long time..{nl}I think the prayer to the goddess was not meaningless..
-QUEST_LV_0300_20160310_001274	Now, the main culprit is the priest of the religious body of the truth, Dominicas..{nl}Everything will be over when we drive him out.{nl}
-QUEST_LV_0300_20160310_001275	Please lend me your power little more.{nl}Even if we have ways to get him, the villagers are all seduced and due to long torture, we can't use our power..
-QUEST_LV_0300_20160310_001276	He seems to have some power to control the plants freely.{nl}Be careful since he is not to be underestimated.
-QUEST_LV_0300_20160310_001277	Dominicas.. Ran away��{nl}
-QUEST_LV_0300_20160310_001278	But, it's okay.{nl}It's a big harvest that we drove out the religious body of the truth from Elgos Monastery.{nl}
+QUEST_LV_0300_20160310_001233	I am talking about the scripture from the Tree of Truth sect left by Dominikas.{nl}It turns out that impious stuff is still remaining in the monastery.{nl}
+QUEST_LV_0300_20160310_001234	If you see the fake scripture, could you bring it to me?{nl}We are short-handed to deal with it individually...{nl}
+QUEST_LV_0300_20160310_001235	Those at the Tree of Truth sect... polluted the power of the goddess...{nl}This is just blasphemy to the goddess!
+QUEST_LV_0300_20160310_001236	So you've found the fake scripture. Thank you.{nl}Give it to me and I will take care of it.
+QUEST_LV_0300_20160310_001237	S...Stop!{nl}I will do whatever you ask me, so please stop!
+QUEST_LV_0300_20160310_001238	Should I just join the Tree of Truth sect?{nl}If so, would I be able to be relieved from this hunger?
+QUEST_LV_0300_20160310_001239	Lies...{nl}If not... How... The black wall...
+QUEST_LV_0300_20160310_001240	It seems he is suffering undernourishment for a long time.{nl}I can't talk to him now.
+QUEST_LV_0300_20160310_001241	Thank you so much...{nl}I am so sorry for misunderstanding you. I thought something was going wrong...{nl}
+QUEST_LV_0300_20160310_001242	After I gained my consciousness, I felt this mysterious energy from you.{nl}What happened?{nl}
+QUEST_LV_0300_20160310_001243	Ah... So you are the Revelator.{nl}I guess this is why the black wall couldn't block you.
+QUEST_LV_0300_20160310_001244	Thank you. I was able to survive because of you.{nl}Dominikas... He is more cruel than the demons.
```

Comment:
"Dominikas... He's worse than the demons."  or "He's more cruel than the demons."